fp-proj/src/app/Modules/shared-quicklink-module/shared-quicklink-module.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
```

Comment:
po co ten modul? jesli ta bibliteka jest wykorzystywana tylko w app routing module to wystarczy go wlasnie tam zaimportowac.